metricbeat/module/kubernetes/types.go

```diff
@@ -134,3 +136,6 @@ type Summary struct {
 		} `json:"volume"`
 	} `json:"pods"`
 }
+
+// PodContainerStatus data
+type PodContainerStatus = v1.ContainerStatus
```

Comment:
Why we need to declare it again? Can't we re-use the already existent https://github.com/elastic/beats/blob/cf1c78c85dda1fd0c3d2e072efd52b903da2cdb0/libbeat/common/kubernetes/types.go#L62-L61 ?

Reply:
You are right. It is not needed